index.bs

```diff
@@ -4969,7 +4969,7 @@ lists and names some [=authenticator types=] of particular interest.
                 <td> [=Multi-factor capable=] </td>
             </tr>
             <tr>
-                <th> <dfn>Passkey platform authenticator</dfn> </th>
+                <th> <dfn>[=Passkey=] platform authenticator</dfn> </th>
```

Comment:
Huh. This looks... odd to me. I rather think we should reference the term in the text. Maybe a bit of rejigging the text overall is overdue at this point, and as part of that I think we should rename "first-factor roaming authenticator" to "passkey roaming authenticator" for symmetry with the platform variant. I took a stab at what that might look like, and came up with meta-PR #2140. What do you think?

Reply:
Changes make sense to me.